pvlib/tests/iotools/test_tmy.py

```diff
@@ -2,13 +2,15 @@
 import pandas as pd
 from pvlib.iotools import tmy
 from ..conftest import DATA_DIR
+import pytest
 
 # test the API works
 from pvlib.iotools import read_tmy3
 
 TMY3_TESTFILE = DATA_DIR / '703165TY.csv'
 TMY2_TESTFILE = DATA_DIR / '12839.tm2'
 TMY3_FEB_LEAPYEAR = DATA_DIR / '723170TYA.CSV'
```

Comment:
I would add a comment here identifying the source of these files as the NSRDB, because the NSRDB TMY3 format is different from SolarAnywhere's.

Reply:
I've added comments to the tests and an admonition in the documentation:
![image](https://user-images.githubusercontent.com/39184289/180495645-da77a05c-b24a-40f3-bd3e-ed9c22c799d3.png)